train_segmentation.py

```diff
@@ -59,26 +61,38 @@ def create_dataloader(samples_folder: Path,
     :param gpu_devices_dict: (dict) dictionary where each key contains an available GPU with its ram info stored as value
     :param sample_size: (int) size of hdf5 samples (used to evaluate eval batch-size)
     :param dontcare_val: (int) value in label to be ignored during loss calculation
+    :param crop_size: (int) size of one side of the square crop performed on original patch during training
     :param num_bands: (int) number of bands in imagery
+    :param min_annot_perc: (int) minimum proportion of ground truth containing non-background information
+    :param attr_vals: (Sequence)
     :param scale: (List) imagery data will be scaled to this min and max value (ex.: 0 to 1)
     :param cfg: (dict) Parameters found in the yaml config file.
+    :param eval_batch_size: (int) Batch size for evaluation (val and test). Optional, calculated automatically if omitted
     :param dontcare2backgr: (bool) if True, all dontcare values in label will be replaced with 0 (background value)
                             before training
     :return: trn_dataloader, val_dataloader, tst_dataloader
     """
     if not samples_folder.is_dir():
-        raise logging.critical(FileNotFoundError(f'\nCould not locate: {samples_folder}'))
-    if not len([f for f in samples_folder.glob('**/*.hdf5')]) >= 1:
-        raise logging.critical(FileNotFoundError(f"\nCouldn't locate .hdf5 files in {samples_folder}"))
-    num_samples, samples_weight = get_num_samples(samples_path=samples_folder, params=cfg)
+        raise FileNotFoundError(f'Could not locate: {samples_folder}')
+    experiment_name = samples_folder.parent.stem
```

Comment:
this code returns incorrect project name "patches" when expected is "template_project"
thus, the following function get_num_samples() can't properly identify *.csv files with tile paths.

it can be fixed like this:
`experiment_name = samples_folder.stem`

Reply:
I'll investigate. Thanks.